media/css/firefox/os/firefox-os-2-5.less

```diff
@@ -3,6 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 @import "../../sandstone/lib.less";
+@import "in-the-news.less";
```

Comment:
Any specific reason to use an `@import` here instead of including the file in the page bundle?